packages/sdks/web-js-sdk/src/sdk/fedcm.ts

```diff
@@ -100,6 +100,9 @@ type OneTapInitialize = ({
 
 type PromptNotification = {
   isSkippedMoment: () => boolean;
+  isDismissedMoment: () => boolean;
```

Comment:
should we document this?

Reply:
@asafshen we should document it, but I don't think that documentation resides in the SDK right? 

This PR will open a doc issue when merged which I will take care of.

Reply:
I'm not sure what is the case for fedCM, but usually we document things in the SDK as well
as a developer I prefer github readme docs and not product website docs, but I see that here we don't have it anyway so probably a different concern 👍